x/staking/spec/01_state.md

```diff
@@ -77,60 +70,23 @@ is updated during the validator set update process which takes place in [`EndBlo
 
 Each validator's state is stored in a `Validator` struct:
 
-```go
-type Validator struct {
-    OperatorAddress         sdk.ValAddress  // address of the validator's operator; bech encoded in JSON
```

Comment:
what I did basically is:
- put all these comments from the old .md files as comments into the .proto files
- then link the proto files directly